src/Command/Wishlist/ImportWishlistFromCsv.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file was created by developers working at BitBag
+ * Do you need more information about us and what we do? Visit our https://bitbag.io website!
+ * We are hiring developers from all over the world. Join us and start your new, exciting adventure and become part of us: https://bitbag.io/career
+*/
+
+declare(strict_types=1);
+
+namespace BitBag\SyliusWishlistPlugin\Command\Wishlist;
+
+use Symfony\Component\HttpFoundation\File\File;
+use Symfony\Component\HttpFoundation\Request;
+
+final class ImportWishlistFromCsv
+{
+    private File $file;
+
+    private Request $request;
+
+    public function __construct(File $file, Request $request)
```

Comment:
SplFileInfo will be more portable.

Reply:
SplFileInfo doesn't provide option to check file MIME type